src/ComponentBase.ts

```diff
@@ -396,18 +378,44 @@ export abstract class ComponentBase<P extends {}, S extends _.Dictionary<any>> e
         return false;
     }
 
+    private _findMatchingSubscription(subscriptions: Dictionary<StoreSubscriptionInternal<P, S>>): boolean {
+        const subscriptionsKeys = Object.keys(subscriptions);
+
+        for (let i = 0, len = subscriptionsKeys.length; i < len; i++) {
```

Comment:
Why use `for` loop here instead of `find` that was being used before?

Reply:
`_.find` handles `Arrays` and `Objects`. `.find` works only with arrays. I added a new version with `values` + `find`.